Userguide/Installation/manualInstallation.md

```diff
@@ -47,46 +49,36 @@ a2enmod proxy_ajp
+Clone the following repo's, it is recommended to create a ```Sources/OSGP``` directory in ```/home/dev/``` since some scripts contain hard coded references to those folders.
 ```shell
 git clone https://github.com/OSGP/Config.git /home/dev/Sources/OSGP/Config
-git clone https://github.com/OSGP/Shared.git /home/dev/Sources/OSGP/Shared
-git clone https://github.com/OSGP/Platform.git /home/dev/Sources/OSGP/Platform
-git clone https://github.com/OSGP/Protocol-Adapter-OSLP.git /home/dev/Sources/OSGP/Protocol-Adapter-OSLP
-git clone https://github.com/OSGP/Protocol-Adapter-DLMS.git /home/dev/Sources/OSGP/Protocol-Adapter-DLMS
-git clone https://github.com/OSGP/Protocol-Adapter-IEC61850.git /home/dev/Sources/OSGP/Protocol-Adapter-IEC61850
-git clone https://github.com/OSGP/Integration-Tests.git /home/dev/Sources/OSGP/Integration-Tests
+git clone https://github.com/OSGP/open-smart-grid-platform.git /home/dev/Sources/OSGP/open-smart-grid-platform
 git clone https://github.com/OSGP/Documentation.git /home/dev/Sources/OSGP/Documentation
 ```
 Make sure you are on the development branch (default).
 
-Initialize the Git submodules in Platform, Protocol-Adapter-OSLP, Protocol-Adapter-DLMS, Protocol-Adapter-IEC61850 and Integration-Tests by running the following command in each directory:
-```shell
-git submodule update --init --recursive
-```
-
 ### Creating directories and symlinks
 Create the following directories:
 - ```/var/log/osp/logs```
 - ```/etc/osp/```
 
-Make the dev user (or equivelant) the owner of the log directory with rwx permission. Give the other users read and execute permission.
+Make the dev user (or equivalent) the owner of the log directory with rwx permission. Give the other users read and execute permission.
 
 Execute the script ```/home/dev/Sources/OSGP/Config/scripts/create-symlinks.sh```
 
 ****Note**** This script uses hard coded references to ```/home/dev/Sources/OSGP/*```, if you used a different user, please edit the script before executing it.
 
-The script will make symlinks to certificates, and to the Maven settings file in ~/.m2
+The script will make symlinks to certificates, to Apache2 configuration and copy configuration settings as samples to locations where theses properties may be overridden.
 
 ### Initiating the Database
 To create the database run the following command (Change /home/dev/ in case of no dev user)
 ```shell
 sudo -u postgres /usr/bin/psql -p 5432 -f /home/dev/Sources/OSGP/Config/sql/create-users-and-databases.sql
 ```
 
-And create a backup of the pg_hba.conf file
+And create a backup of the pg_hba.conf file (modify if your version of PostgreSQL is different)
 ```shell
-cp -p /etc/postgresql/9.3/main/pg_hba.conf /etc/postgresql/9.3/main/pg_hba.backup
+cp -p /etc/postgresql/9.4/main/pg_hba.conf /etc/postgresql/9.4/main/pg_hba.backup
```

Comment:
I think the reason I chose 9.4 was because of the link to the postgresql-9.4.1209.jre7.jar that was present on the manual installation document before. This link was updated as well however.
As discussed, it looks better with a version number here (as opposed to leaving it out) since the referenced file path includes the version number.
To prevent having to update this page for each new PostgreSQL version, the old version remains with the note that is should actually be a version matching the installation.